.asf.yaml

```diff
@@ -0,0 +1,42 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features
+
+notifications:
+  commits: commits@arrow.apache.org
+  issues: issues@arrow.apache.org
+  pullrequests: github@arrow.apache.org
+
+github:
+  description: "Apache Arrow Julia"
+  homepage: https://arrow.apache.org/
+  labels:
+    - apache-arrow
+    - julia
+  features:
+    issues: true
+  enabled_merge_buttons:
+    squash: true
+    merge: false
+    rebase: false
```

Comment:
@quinnj It seems that JuliaData/Arrow.jl uses only squash merge.

Reply:
We allowed squash and rebase, but not merge.

Reply:
OK. I've enabled rebase.